dask/_expr.py

```diff
@@ -462,7 +471,9 @@ def _funcname(self) -> str:
 
     @functools.cached_property
     def _name(self) -> str:
-        return self._funcname + "-" + _tokenize_deterministic(*self.operands)
+        if not self._token:
```

Comment:
The name is overwritten in a few places, is this an issue?

Reply:
Maybe. I'll check

Reply:
yeah, that's an issue

Reply:
there are just 38 places this is overridden one way or another... no big deal!